.github/workflows/python.yml

```diff
@@ -226,10 +226,10 @@ jobs:
       PYTEST_SENTRY_ALWAYS_REPORT: no
 
       # services configuration
-      SENTRY_REDIS_HOST: redis
+      SENTRY_REDIS_HOST: sentry_redis
       DATABASE_URL: postgresql://postgres:postgres@localhost/sentry
-      SENTRY_KAFKA_HOSTS: localhost:9092
-      SENTRY_ZOOKEEPER_HOSTS: localhost:2181
+      SENTRY_KAFKA_HOSTS: 127.0.0.1:9092
+      SENTRY_ZOOKEEPER_HOSTS: sentry_zookeeper:2181
```

Comment:
```suggestion
      SENTRY_ZOOKEEPER_HOSTS: 127.0.0.1:2181
```

And in other areas.

Reply:
@joshuarli This doesn't work :< https://github.com/getsentry/sentry/pull/20986/checks?check_run_id=1167768037#step:11:1092

Reply:
Hmmmm, weird, let me try and understand why first.